psydac/api/tests/test_2d_multipatch_mapping_poisson.py

```diff
@@ -332,10 +316,10 @@ def test_poisson_2d_2_patches_dirichlet_parallel_0():
 
     D1     = mapping_1(A)
     D2     = mapping_2(B)
-
-    domain = D1.join(D2, name = 'domain',
-                bnd_minus = D1.get_boundary(axis=1, ext=1),
-                bnd_plus  = D2.get_boundary(axis=1, ext=-1))
+                
+    connectivity = [((0,1,1),(1,1,-1))]
+    domains = [D1, D2]
+    domain = Domain.join(domains, connectivity, 'domain')
```

Comment:
Can you please rename `domains` as `patches`?

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)